publisher/publisher.go

```diff
@@ -1,31 +1,28 @@
 package publisher
 
 import (
-	"bufio"
 	"fmt"
-	"io"
 	"io/ioutil"
-	"math/rand"
-	"runtime"
-	"strings"
+	"os"
 	"time"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/honeycombio/dynsampler-go"
 	"github.com/honeycombio/honeyelb/options"
+	"github.com/honeycombio/honeyelb/state"
 	"github.com/honeycombio/honeytail/event"
-	"github.com/honeycombio/honeytail/parsers/nginx"
 	"github.com/honeycombio/libhoney-go"
 	"github.com/honeycombio/urlshaper"
 )
 
 const (
 	AWSElasticLoadBalancerFormat = "aws_elb"
+	AWSCloudFrontWebFormat       = "aws_cf_web"
 )
 
 var (
 	// 2017-07-31T20:30:57.975041Z spline_reticulation_lb 10.11.12.13:47882 10.3.47.87:8080 0.000021 0.010962 0.000016 200 200 766 17 "PUT https://api.simulation.io:443/reticulate/spline/1 HTTP/1.1" "libhoney-go/1.3.3" ECDHE-RSA-AES128-GCM-SHA256 TLSv1.2
-	logFormat           = []byte(fmt.Sprintf(`log_format %s '$timestamp $elb $client_authority $backend_authority $request_processing_time $backend_processing_time $response_processing_time $elb_status_code $backend_status_code $received_bytes $sent_bytes "$request" "$user_agent" $ssl_cipher $ssl_protocol';`, AWSElasticLoadBalancerFormat))
+	logFormat = []byte(fmt.Sprintf(`log_format %s '$timestamp $elb $client_authority $backend_authority $request_processing_time $backend_processing_time $response_processing_time $elb_status_code $backend_status_code $received_bytes $sent_bytes "$request" "$user_agent" $ssl_cipher $ssl_protocol';
```

Comment:
please clarify these more. A super super long line (or pair of lines, all in an outer quote) with two constants at the end isn't the most readable.  Also an example line for cloudfront (to mirror the ELB reticulating example) would be lovely.